packages/ckeditor5-core/src/plugincollection.js

```diff
@@ -206,12 +206,12 @@ export default class PluginCollection {
 			 * @error plugincollection-plugin-not-found
 			 * @param {Array.<String>} plugins The name of the plugins which could not be loaded.
 			 */
-			const errorMsg = 'plugincollection-plugin-not-found: Some plugins are not available and could not be loaded.';
+			const errorId = 'plugincollection-plugin-not-found';
```

Comment:
One more error to fix: https://github.com/ckeditor/ckeditor5/blob/b3a0bba/packages/ckeditor5-core/src/plugincollection.js#L134

Reply:
@jodator Maybe the tests should also expect one space after the `errorName`? So that line https://github.com/ckeditor/ckeditor5/blob/14007d342bdef02009425c059af4308c4012eb17/packages/ckeditor5-core/tests/plugincollection.js#L524 becomes:

```
					/^plugincollection-plugin-not-loaded /, editor, { plugin: 'foo' }
```

Reply:
I'd go with those changes in a follow-up: #8140. Let's fix it once and in all places.

Fixing one case is not worth the effor IMO.